channeldb/payments.go

```diff
@@ -957,6 +927,78 @@ func (d *DB) DeletePayments(failedOnly, failedHtlcsOnly bool) error {
 	}, func() {})
 }
 
+// deletePaymentAttempt removes an HTLC attempt from disk and returns the
+// payment in the state before it was deleted.
+func (d *DB) deletePaymentAttempt(hash lntypes.Hash, attemptID uint64) (*MPPayment, error) {
```

Comment:
nit: line too long (exceeds the 80 limit)

Reply:
Done.